src/neptune/exceptions.py

```diff
@@ -823,15 +823,23 @@ def __init__(self, reason: str):
 {end}
 {reason}
 
-It's not possible to upload new data, but you can still fetch and delete data.
-If you are using asynchronous (default) connection mode, Neptune automatically switched to offline mode
-and your data is being stored safely on the disk. You can upload it later using the Neptune Command Line Interface tool:
-    {bash}neptune sync -p project_name{end}
-What should I do?
-    - In case of storage limitations, go to your projects and remove runs or model metadata you don't need
-    - ... or update your subscription plan here: https://app.neptune.ai/-/subscription
-You may also want to check the following docs page:
+Data synchronization was interrupted because you're out of storage space. If you're using Neptune in asynchronous mode
+(default), the data that couldn't be uploaded is safely stored on the disk. You can still fetch and delete data from
+your project(s).
+How can I free up space in Neptune?
+    - Contact your workspace admin about possible project or workspace storage limits.
```

Comment:
For the record: IMO we can live with this in the client until the feature is released in full.